src/Database/Log/QueryLogger.php

```diff
@@ -46,7 +46,7 @@ public function log($level, $message, array $context = [])
     {
         Log::write(
             'debug',
-            (string)$context['query'],
+            'connection=' . $this->getConfig('connection', '') . ' ' . (string)$context['query'],
             ['scope' => $this->scopes() ?: ['queriesLog']]
```

Comment:
Should we be adding the connection name here, and using the new log formatting features that replace `{foo}` expressions to format the connection name into the message?

Having the connection name as a context value would allow structured logging adapters to use the connection name as data instead of text.

Reply:
I looked at adding it as a context value. The `LoggingStatement` would need the connection name to add it. Can we do it another way?

Reply:
You can  `$this->_driver->getConnection()->getName()` to get the connection name directly in `LoggingStatement`.

Reply:
Should we just provide the full connection object in the context then? Users can add any custom info they want to the connection like getting/setting connection ids.

Reply:
My mistake, `$this->_driver->getConnection()` return the driver's `PDO` instance. So without modifying the `LoggingStatement` there's not way to pass the connection name to the logger.

Reply:
I'm ok with the constructor argument to QueryLogger, my suggestion was that instead of appending the string here, we pass the connection as a context value. I can do that in a follow up pull request though, as it would be good to move duration and numRows into context to for consistency.

Reply:
> ..it would be good to move duration and numRows into context to for consistency.

Duration and numRows are already available through context since the last round of cleanup I did for query logging. Just they are not set directly set as context keys but available through the `LoggedQuery` instance which is passed to context.